### PR TITLE
Adds the ability to change base URI of scrapers

### PR DESCRIPTION
**Pull request by tlm**
Adds the ability to change the base uri for scrapers. Use case if for restricted environments.

### CODE REVIEW

**Review by GKTheOne — APPROVED** [review]

File: src/main/resources/application-rdap.yml [hunk 1]
```diff
@@ -58,7 +58,8 @@ rdap:
     scraping:
         scrapers:
             # Individual scrapers available in rdap-ingressd can be enabled and
-            # disabled.
+            # disabled. It's also possible to speficify the property baseURI for
```
Comment (khloke) [inline src/main/resources/application-rdap.yml]: *specify

File: src/main/java/net/apnic/rdap/nro/config/NROConfiguration.java [hunk 2]
```diff
@@ -4,18 +4,36 @@
...
+    public String getBaseURI()
+    {
+        return baseURI;
+    }
+
+    @ConditionalOnProperty(value="rdap.scraping.scrapers.nro.enabled")
     @Bean(value="nro")
     public NROScraper nroScraper()
     {
+        if(baseURI != null)
```
Comment (khloke) [inline src/main/java/net/apnic/rdap/nro/config/NROConfiguration.java]: (Suggestion) Is it possible for this to be an empty string? Might be worth checking `!baseURI.isEmpty()`.

File: src/main/java/net/apnic/rdap/iana/config/IANAConfiguration.java [hunk 3]
```diff
@@ -4,19 +4,37 @@
...
+    public String getBaseURI()
+    {
+        return baseURI;
+    }
+
     @ConditionalOnProperty(value="rdap.scraping.scrapers.iana.enabled")
     @Bean(value="iana")
     public IANABootstrapScraper ianaScraper()
     {
+        if(baseURI != null)
```
Comment (khloke) [inline src/main/java/net/apnic/rdap/iana/config/IANAConfiguration.java]: (Suggestion) Again, might be worth checking for empty string.

File: src/main/resources/application-rdap.yml (hunk 1 above)
Comment (tlm) [reply src/main/resources/application-rdap.yml]: Done & Done

File: src/main/java/net/apnic/rdap/nro/config/NROConfiguration.java (hunk 2 above)
Comment (tlm) [reply src/main/java/net/apnic/rdap/nro/config/NROConfiguration.java]: Done & Done

File: src/main/java/net/apnic/rdap/iana/config/IANAConfiguration.java (hunk 3 above)
Comment (tlm) [reply src/main/java/net/apnic/rdap/iana/config/IANAConfiguration.java]: Done & Done

**Pull request merged**